### PR TITLE
Able to search without prefix "0x"

### PR DESCRIPTION
**Pull request by saneery**
Resolves https://github.com/poanetwork/blockscout/issues/1580

### CODE REVIEW

File: apps/block_scout_web/lib/block_scout_web/chain.ex [hunk 1]
```diff
@@ -60,13 +60,17 @@ defmodule BlockScoutWeb.Chain do
...
   def from_param(param)
 
-  def from_param("0x" <> number_string = param) do
-    case String.length(number_string) do
-      40 -> address_from_param(param)
-      64 -> block_or_transaction_from_param(param)
-      _ -> {:error, :not_found}
-    end
-  end
+  def from_param("0x" <> number_string = param) when byte_size(number_string) == 40,
```
Comment (vbaranov) [inline apps/block_scout_web/lib/block_scout_web/chain.ex]: @saneery does it make sense to define 40, 64 as constants in the module?

Comment (saneery) [reply apps/block_scout_web/lib/block_scout_web/chain.ex]: Yes, good idea, thanks!

**Comment by saneery** [comment]
@vbaranov I updated the PR, can we merge it?

**Comment by vbaranov** [comment]
>I updated the PR, can we merge it?

Yes, I will do. Thank you!

**Pull request merged**